src/Servers/HttpSys/src/RequestProcessing/ResponseBody.cs

```diff
@@ -144,28 +144,21 @@ private unsafe void FlushInternal(bool endOfRequest, ArraySegment<byte> data = n
         Span<GCHandle> pinnedBuffers = default;
         try
         {
-            Span<HttpApiTypes.HTTP_DATA_CHUNK> dataChunks;
-            BuildDataChunks(ref allocator, endOfRequest, data, out dataChunks, out pinnedBuffers);
+            BuildDataChunks(ref allocator, endOfRequest, data, out var dataChunks, out pinnedBuffers);
             if (!started)
             {
                 statusCode = _requestContext.Response.SendHeaders(ref allocator, dataChunks, null, flags, false);
             }
             else
             {
-                fixed (HttpApiTypes.HTTP_DATA_CHUNK* pDataChunks = dataChunks)
```

Comment:
This is because `dataChunks` comes from the unmanaged heap right? We just forgot to remove `fixed`

Reply:
No, the fixed is now part of the generated code.